templates/pioneer-meta/pioneer-enrollment/pioneer-enrollment.1.schema.json

```diff
@@ -0,0 +1,8 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "pioneer-enrollment",
+  "$comment": "Bug 1663857 - Add pioneer-enrollment pings for explict consent logging.",
+  "properties": {},
+  "additionalProperties": false
```

Comment:
Why aren't we giving these pings more explicit structure? It looks like we have a validation example with a well-defined format.

Reply:
`"additionalProperties": false` with an empty object schema is the most explicit schema for an empty document. The validation document is the envelope, this is the actual message. It's being treated like the deletion ping. See https://github.com/mozilla/gcp-ingestion/pull/1394 for more context.